pkg/cmd/auth/login/login.go

```diff
@@ -69,9 +69,10 @@ func NewCmdLogin(f *cmdutil.Factory, runF func(*LoginOptions) error) *cobra.Comm
 			stored location.
 
 			Alternatively, use %[1]s--with-token%[1]s to pass in a personal access token (classic) on standard input.
-			The minimum required scopes for the token are: %[1]srepo%[1]s, %[1]sread:org%[1]s, and %[1]sgist%[1]s.
-
-			Fine-grained personal access tokens are not supported.
+			The minimum required scopes for the token are: %[1]srepo%[1]s, %[1]sread:org%[1]s, and %[1]sgist%[1]s. Although
+			it is possible to pass a fine-grained personal access token to %[1]s--with-token%[1]s, it should be done with
+			care, as the inherent scoping to certain resources may cause confusing behaviour when interacting with other
+			resources. Favour setting %[1]sGH_TOKEN$%[1]s for fine-grained personal access token usage. 
```

Comment:
I appreciate the fine line being walked here:

1. FGPAT are supported in the general sense
2. Understanding all of the fine-grained permissions and how they relate to the various commands has not been done nor is it documented

That said, are we committing to resolving any bug raised regarding FGPAT behavior given they are still in public preview?

Reply:
Minor nit that isn't the end of the world, but simplifying the language a little by warning users upfront in case someone only reads the first part and not the latter:

```suggestion
			The minimum required scopes for the token are: %[1]srepo%[1]s, %[1]sread:org%[1]s, and %[1]sgist%[1]s.
			Take care when passing a fine-grained personal access token to %[1]s--with-token%[1]s
			as the inherent scoping to certain resources may cause confusing behaviour when interacting with other
			resources. Favour setting %[1]sGH_TOKEN$%[1]s for fine-grained personal access token usage. 
```

Reply:
> That said, are we committing to resolving any bug raised regarding FGPAT behavior given they are still in public preview?

I dunno. I think my position is that we will try to support any use of fine-grained tokens that currently works in the API, and we will make some attempts towards making life more pleasant for our users in the cases they don't. In general it feels like "the right thing to do" to help our users user safer options.

It also seems like GitHub the business has tokens in public preview (i.e. more than just the team that owns them) and so we would want to support them as if we owned them.

Reply:
In practice it's hard to imagine Fine Grained PATs being walked back in any fundamental way. They have been in public preview for so long, I imagine there is a lot of reliance on them.